CHANGELOG.md

```diff
@@ -28,6 +28,26 @@ CHANGELOG
 Swift Next
 ----------
 
+* [SE-0299][]:
+
+  It is now possible to use leading-dot syntax in generic contexts to access static members of protocol extensions where `Self` is constrained to a concrete type:
+
+  ```swift
+  public protocol ToggleStyle { ... }
+
+  public struct DefaultToggleStyle: ToggleStyle { ... }
+
+  extension ToggleStyle where Self == DefaultToggleStyle {
```

Comment:
Perhaps we should mention that the type must be fully concrete (I suppose `Self == Foo<Assoc>` will not work).

Reply:
✅